src/journal/models.py

```diff
@@ -90,10 +91,21 @@ class Journal(AbstractSiteModel):
     enable_correspondence_authors = models.BooleanField(default=True)
     disable_html_downloads = models.BooleanField(default=False)
     full_width_navbar = models.BooleanField(default=False)
-    is_remote = models.BooleanField(default=False)
+    is_remote = models.BooleanField(
+        default=False,
+        help_text='When enabled the journal is marked as not hosted in Janeway.',
```

Comment:
Shall we make all these new help_texts translatable?

Also, could use a comma: "When enabled, the journal..."